### PR TITLE
Introduce option to globally disable format checks

### LINKED ISSUE #120: Introduce global option to disable all format checks

### PR DESCRIPTION
**Pull request by The-Huginn**
Resolves #120

### CODE REVIEW

File: src/test/java/io/xstefank/wildfly/bot/PRCommitCheckTest.java [hunk 1]
```diff
@@ -184,4 +188,23 @@ public void testOverridingCommitMessage() throws IOException {
...
+                """;
+        mockedContext = MockedContext.builder(pullRequestJson.id())
+                .commit(INVALID_COMMIT_MESSAGE);
+        given().github(mocks -> Util.mockRepo(mocks, wildflyConfigFile, pullRequestJson))
+                .when().payloadFromString(pullRequestJson.jsonString())
+                .event(GHEvent.PULL_REQUEST)
+                .then().github(mocks -> {
+                    GHRepository repo = mocks.repository(TEST_REPO);
+                    Mockito.verify(repo, never()).createCommitStatus(anyString(), any(), anyString(), anyString());
+                    Mockito.verify(repo, never()).createCommitStatus(anyString(), any(), anyString(), anyString(), anyString());
```
Comment (xstefank) [inline src/test/java/io/xstefank/wildfly/bot/PRCommitCheckTest.java:207]: probably doesn't need to verify "never" two times

**Review by xstefank — COMMENTED** [review]
Please fix the tests in a follow up

File: src/test/java/io/xstefank/wildfly/bot/PRCommitCheckTest.java (hunk 1 above)
Comment (The-Huginn) [reply src/test/java/io/xstefank/wildfly/bot/PRCommitCheckTest.java:207]: I verify that neither of the methods is called, as it's overloaded method

Comment (xstefank) [reply src/test/java/io/xstefank/wildfly/bot/PRCommitCheckTest.java:207]: ok, I missed the number of arguments. ok then

**Pull request merged**